app/uk/gov/hmrc/transitmovementsrouter/connectors/EISConnector.scala

```diff
@@ -140,7 +142,15 @@ class EISConnectorImpl(
               logger.info(logMessage + s"Response status: ${result.status}")
               Right(())
             case Left(error) =>
-              logger.warn(logMessage + s"Response status: ${error.statusCode}")
+              val status =
+                if (logBodyOn500 && error.statusCode == INTERNAL_SERVER_ERROR) {
+                  s"""Response status: ${error.statusCode}"
```

Comment:
Does the logged message contain identifying personal info?

Reply:
That's what I'm slightly concerned about, I want to see what comes back from QA and make a decision as to whether to turn it off from there (via the config).